### PR TITLE
Dmitrii/feature/api calls

### PR DESCRIPTION
**Pull request by dgubko**
This branch sets up the necessary api calls and data cleaning, incorporates an initial data typing.

### CODE REVIEW

**Review by knpausch — APPROVED** [review]
Great work driving for us today Dimitri

**Review by Nagel29 — APPROVED** [review]
Great start!

**Review by Courtney-Lippman — APPROVED** [review]
This all looks good. This is an excellent start to the project!

File: src/utilities/utilities.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    rank: number;
+    id: string;
+    price: string;
+    average_user_rating: number;
+    num_user_ratings: number;
+    description: string;
+    players: string;
+    officialUrl: string;
+    playtime: string;
+    image: string;
```
Comment (Adelle-Pitsas) [inline src/utilities/utilities.ts:37]: This is a good place where we can extend the Game interface: `interface Details extends Game { the types we are adding to Game }`

**Comment by Adelle-Pitsas** [comment]
Looks great team!

**Pull request merged**